pkg/rancher-desktop/pages/Containers.vue

```diff
@@ -284,12 +284,15 @@ export default defineComponent({
         }));
     },
     errorMessage() {
-      if (this.errorMessage) {
-        return this.errorMessage;
+      if (this.execError) {
+        return this.execError;
```

Comment:
This is fixes an issue where once an error occurs, the message would never go away because we checked the error getter for the custom message, so it would just always return the previous error…